backend/de.metas.business/src/main/java/org/adempiere/mm/attributes/api/IAttributeSetInstanceBL.java

```diff
@@ -87,6 +87,8 @@ default I_M_AttributeSetInstance createASIFromAttributeSet(IAttributeSet attribu
 		return createASIFromAttributeSet(attributeSet, Predicates.alwaysTrue());
 	}
 
+	void cloneOrCreateASI(Object to, Object from);
```

Comment:
```suggestion
	/**
	  * Clone the ASI from the given {@code from} to the given {@code to}, or just create a new create empty ASI in {@code to} in case from is null or "none"
	  */
	void cloneOrCreateASI(Object to, Object from);
```